manager2/src/app/project/project.component.html

```diff
@@ -18,9 +18,15 @@ <h3>Request a new project creation</h3>
           </div>
 
           <div class="form-group row col-sm-8">
-            <label for="new_project_size">Size (GB)</label>
-            <input placeholder="project size in G" type="number" id="new_project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
-            <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
+              <label for="new_project_size">Size (GB)</label>
+              <input placeholder="project size in GB" type="number" id="new_project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+              <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
+          </div>
+
+          <div class="form-group row col-sm-8">
+              <label for="new_project_cpu">Cpu (Hour)</label>
+              <input placeholder="project cpu in G" type="number" id="new_project_cpu" [value]="default_cpu" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.cpu" class="form-control"/>
```

Comment:
Cpu in g again

Reply:
fixed: 5ecb403008f78cc3716119dfae809234dba93363